docs/sources/auth/azuread.md

```diff
@@ -24,26 +24,31 @@ To enable the Azure AD OAuth2 you must register your application with Azure AD.
 
 1. Under **Manage** in the side menu, click **App Registrations** and then **New Registration**. Provide a fitting name.
 
-1. Under **Redirect URI**, select **Web** as the app type. 
+1. Under **Redirect URI**, select **Web** as the app type.
 
 1. Add the redirect URL `https://<grafana domain>/login/azuread`, then click **Register**.
 
-1. The app's **Overview** page is displayed. Note the **Application ID**, this is the OAuth client id.
+1. The app's **Overview** page is displayed. Note the **Application ID**. This is the OAuth client ID.
 
 1. Click **Endpoints** from the top menu.
 
-   - Note the **OAuth 2.0 authorization endpoint (v2)**, this is the auth URL.
-   - Note the **OAuth 2.0 token endpoint (v2)**, this is the token URL.
+   - Note the **OAuth 2.0 authorization endpoint (v2)**. This is the auth URL.
+   - Note the **OAuth 2.0 token endpoint (v2)**. This is the token URL.
 
-1. Click **Certificates & secrets** and add a new entry under Client secrets.
-    - Description: Grafana OAuth
-    - Expires: Never
+1. Click **Certificates & secrets** and then add a new entry under **Client secrets**.
 
-1. Click **Add**, then copy the key value. This is the OAuth client secret.
+   - Description: Grafana OAuth
+   - Expires: Never
+
+1. Click **Add**, and then copy the key value. This is the OAuth client secret.
 
 1. Click **Manifest**.
-   - Add definitions for the required Application Roles for Grafana (Viewer, Editor, Admin). Without this configuration, all users will be assigned the Viewer role.
-   - Every role requires a unique id. On Linux, this can be created with `uuidgen`. For example:
+
+   - Define the required Application Role values for Grafana: Viewer, Editor, Admin. Without this configuration, all users receive the Viewer role.
```

Comment:
```suggestion
   - Define the required Application Role values for Grafana: Viewer, Editor, Admin. Otherwise, all users will have the Viewer role.
```